### PR TITLE
Add the addShould function to ProxyQuery

### PR DESCRIPTION
**Pull request by greg0ire**
The addShould function was missing

### CODE REVIEW

File: ProxyQuery/ElasticaProxyQuery.php [hunk 1]
```diff
@@ -190,6 +190,19 @@ public function addMustNot($args)
     }
 
     /**
+     * Add should part to query.
+     *
+     * @param \Elastica\Query\AbstractQuery|array $args Should query
```
Comment (soullivaneuh) [inline ProxyQuery/ElasticaProxyQuery.php]: Should be `AbstractQuery` query with the associated `use` key.

File: ProxyQuery/ElasticaProxyQuery.php [hunk 2]
```diff
@@ -190,6 +190,19 @@ public function addMustNot($args)
...
+     *
+     * @return $this
```
Comment (soullivaneuh) [inline ProxyQuery/ElasticaProxyQuery.php]: `@return ElasticaProxyQuery`

File: ProxyQuery/ElasticaProxyQuery.php (hunk 1 above)
Comment (greg0ire) [inline ProxyQuery/ElasticaProxyQuery.php]: You mean without the namespace ?

Comment (greg0ire) [inline ProxyQuery/ElasticaProxyQuery.php]: Assuming you meant that

Comment (soullivaneuh) [inline ProxyQuery/ElasticaProxyQuery.php]: :+1:

**Pull request merged**